apps/myaccount/src/components/overview/overview.tsx

```diff
@@ -44,13 +45,19 @@ export const Overview: FunctionComponent<OverviewPropsInterface> = (
 ): ReactElement => {
 
     const {
-        enableThreeWidgetLayout
+        userSource,
+        enableAlternateWidgetLayout
     } = props;
 
     const accessConfig: FeatureConfigInterface = useSelector((state: AppState) => state?.config?.ui?.features);
     const allowedScopes: string = useSelector((state: AppState) => state?.authenticationInformation?.scope);
 
-    const accountStatus = (widthComputer: SemanticWIDTHS, widthMobile: SemanticWIDTHS): React.ReactElement => {
+    /**
+     * Profile status widget with link to profile.
+     * @param widthComputer
```

Comment:
Check other places as well.

```suggestion
     * @param {SemanticWIDTHS} widthComputer - Width for the computer screens.
```